web/src/rest/publish.ts

```diff
@@ -22,6 +31,8 @@ export function girderize(publishedDandiset: Version) {
     lowerName: name,
     bytes: size,
     items: asset_count,
+    status,
+    validationError: validation_error,
```

Comment:
What's the point of updating `girderize`? Is it just to maintain consistency and avoid removing the function in this PR?

Reply:
Removing `girderize` is definitely not happening in this PR since it would impact everything everywhere that uses versions, so we have to work with it.

There are two new fields, `status` and `validation_error` that are now being returned from the Django API. For those fields to actually appear in the girderized versions that are stored in Vuex, we need to change `girderize` to include those fields.